app/assets/javascripts/browse-columns.js

```diff
@@ -51,6 +61,47 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     }
   }
 
+  BrowseColumns.prototype.fireClickEvent = function (link) {
+    var category = link.getAttribute('data-track-category')
+    var action = link.getAttribute('data-track-action')
+    var options = JSON.parse(link.getAttribute('data-track-options'))
+
+    if (this.displayState === 'root') {
+      options.dimension32 = 'Browse Index'
+      this.rightMostColumn = this.$root
+    } else if (this.displayState === 'section') {
+      options.dimension32 = 'First Level Browse'
+      this.rightMostColumn = this.$section
+    } else if (this.displayState === 'subsection') {
+      this.rightMostColumn = this.$subsection
+      options.dimension32 = 'Second Level Browse'
+    }
+    this.$module.dimension26 = this.getDimension26()
+    this.$module.dimension27 = this.getDimension27()
+    options.dimension26 = this.$module.dimension26
+    options.dimension27 = this.$module.dimension27
+    options.label = link.getAttribute('data-track-label')
+    options.location = document.location.href
+    options.title = document.title.replace('Browse:', '').trim()
+
+    if (window.GOVUK.analytics && window.GOVUK.analytics.trackEvent) {
+      GOVUK.analytics.trackEvent(category, action, options)
+    }
+  }
+
+  // dimension26 should be the number of 'sections' of links in the
+  // right-most visible column. For the first two columns this is
+  // always just 1
+  BrowseColumns.prototype.getDimension26 = function () {
```

Comment:
Could we rename the function to be more reflective of what it does and shorten the comment? Something like sectionLinkCount?